.github/workflows/run_release.yaml

```diff
@@ -9,58 +9,77 @@ on:
       RELEASE_VERSION: ${{ inputs.version }}
     permissions:
+      id-token: write
       contents: write
     steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-      with:
-        path: repo
-    - name: Sign Scripts
-      shell: pwsh
-      run: |
-        $ErrorActionPreference = "Stop"
-        $PSDefaultParameterValues['*:ErrorAction']='Stop'
-        Set-StrictMode -Version Latest
-
-        New-Item -ItemType directory -Path certificate
-        Set-Content -Path certificate\cert.txt -Value $env:CODESIGN_PFX
-        certutil -decode certificate\cert.txt certificate\cert.pfx
-
-        $cert = Get-PfxCertificate -FilePath certificate\cert.pfx -Password (ConvertTo-SecureString -String $env:CODESIGN_PW -Force -AsPlainText)
-
-        Get-ChildItem -Recurse -Path repo -Include **.ps1,**.psm1,**.psd1 | ForEach-Object {
-          $path = $_.FullName
-          Set-AuthenticodeSignature -Certificate $cert -FilePath $path -TimestampServer "http://timestamp.digicert.com/" -IncludeChain NotRoot -HashAlgorithm SHA256
-          # Delay for 4 seconds to avoid exceeding rate limits (1000 / 5 minutes, 100 / 5 seconds)
-          Start-Sleep -Seconds 4
-        }
-
-        Remove-Item -Recurse -Force certificate
-        Remove-Item -Recurse -Force repo -Include .git*
-
-        Move-Item  -Path repo -Destination "ScubaGear-${env:RELEASE_VERSION}" -Force
-        Compress-Archive -Path "ScubaGear-${env:RELEASE_VERSION}" -DestinationPath "ScubaGear-${env:RELEASE_VERSION}.zip"
-
-        Get-ChildItem -Path . | Write-Output
-    - name: release
-      uses: softprops/action-gh-release@v1
-      id: create_release
-      with:
-        draft: true
-        prerelease: false
-        name: ${{ inputs.releaseName }}
-        tag_name: v${{ inputs.version }}
-        files: ScubaGear-${{ inputs.version }}.zip
-        generate_release_notes: true
-        fail_on_unmatched_files: true
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: repo
+      - name: OIDC Login to Azure Public Cloud with AzPowershell (enableAzPSSession true)
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          enable-AzPSSession: true       
+      - name: Sign Module
+        uses: azure/powershell@v1
+        with:
+          inlineScript: |
+            $DebugPreference='Continue'
+            dotnet --version
+            dotnet tool install --global AzureSignTool --version 4.0.1
+            #TODO: Add Key Vault info to environment secret (Issue #790)
+            . repo/utils/DeployUtils.ps1
+            # Remove non-release files
+            Remove-Item -Recurse -Force repo -Include .git*
+            SignScubaGearModule `
+              -AzureKeyVaultUrl https://githubautomationvault.vault.azure.net/ `
```

Comment:
Please parameterize this (e.g. into the secret the way the tenant is parameterized).

Reply:
Ok.  It may be worth just making this change now and testing it in dev, though I'm not going to hold the PR over that.  Definitely need to do it before we roll out production.

Reply:
Okay, will update to add now.  WIll leave #790 to handle production.

Reply:
Added development secret with key vault info.